app/scripts/lib/rpc-method-middleware/handlers/log-web3-usage.js

```diff
@@ -0,0 +1,50 @@
+import { MESSAGE_TYPE } from '../../enums'
```

Comment:
Beings as we are setting future patterns here, maybe we should take this opportunity to include a docblock at the top of the file that describes the general purpose of the middleware. You can keep individual method documentation too... but why does this middleware exist as a whole?